roles/appdeploy/defaults/main.yml

```diff
@@ -1,2 +1,3 @@
 ---
 # defaults file for roles/appdeploy
+desired_version: v16.14.0
```

Comment:
I'd make this `desired_node_version` throughout, because `desired_version` could be anything. Also, add a note that if the project requires a certain version, the value for this should match it. For example, https://github.com/pulibrary/orangelight/blob/0648c94b853a4d1ed61fad48128fa54eaf68317c/.tool-versions#L3.